packages/core/useVirtualList/index.ts

```diff
@@ -42,7 +42,7 @@ export interface UseVirtualListItem<T> {
 }
 
 export interface UseVirtualListReturn<T> {
-  list: Ref<UseVirtualListItem<T>[]>
+  list: Ref<readonly UseVirtualListItem<T>[]>
```

Comment:
this will only return a readonly, no?

![image](https://github.com/user-attachments/assets/39c12410-c6fd-4dfb-9255-1c63e6d84bfe)

Reply:
Ah, I'll fix this up.